pkg/component/networking/vpn/seedserver/seedserver.go

```diff
@@ -310,16 +323,16 @@ func (v *vpnSeedServer) podTemplate(configMap *corev1.ConfigMap, secretCAVPN, se
 							Value: strings.Join(ipFamilies, ","),
 						},
 						{
-							Name:  "SERVICE_NETWORK",
-							Value: v.values.Network.ServiceCIDRs[0].String(),
+							Name:  "SERVICE_NETWORKS",
+							Value: strings.Join(serviceCIDRs, ","),
 						},
 						{
-							Name:  "POD_NETWORK",
-							Value: v.values.Network.PodCIDRs[0].String(),
+							Name:  "POD_NETWORKS",
+							Value: strings.Join(podCIDRs, ","),
 						},
 						{
-							Name:  "NODE_NETWORK",
-							Value: nodeNetwork,
+							Name:  "NODE_NETWORKS",
+							Value: strings.Join(nodeCIDRs, ","),
```

Comment:
While feature gate `NewVPN` exists, we cannot remove the old environment variables, but only need to add the new ones. In case you want a clean implementation here, i.e. only one set of environment variables, you would need to implement the new logic in the "old" vpn.